JASP-Desktop/widgets/listmodelcustomcontrasts.h

```diff
@@ -16,28 +16,35 @@
 // <http://www.gnu.org/licenses/>.
 //
 
-#ifndef LISTMODELMARGINALMEANSCONTRASTS_H
-#define LISTMODELMARGINALMEANSCONTRASTS_H
+#ifndef LISTMODELCUSTOMCONTRASTS_H
```

Comment:
These kinds of things can also be replaced with `#pragma once`

Reply:
I know, but all header files are like that now, and even Qt generates new classes with this kind of header...